.github/workflows/reusable-build.yml

```diff
@@ -86,46 +86,46 @@ jobs:
       working-directory: ${{env.GITHUB_WORKSPACE}}
       run: nuget restore ${{env.SOLUTION_FILE_PATH}}
 
-    - name: Cache verifier project
-      uses: actions/cache@9b0c1fce7a93df8e3bb8926b0d6e9d89e92f20a7
-      env:
-        cache-name: cache-verifier-project
-      with:
-        path: external/ebpf-verifier/build
-        # Increase the version of the cache each time the CXXFLAGS are modified to invalidate the cache.
-        key: ${{ runner.os }}-${{env.BUILD_PLATFORM}}-${{env.BUILD_CONFIGURATION}}-${{env.BUILD_ARTIFACT_NAME}}-${{ hashFiles('.git/modules/external/ebpf-verifier/HEAD') }}-v1
+    # - name: Cache verifier project
```

Comment:
Is there a reason we're keeping this 'dead' (commented out) block? If not, could we remove it?

Reply:
The idea was to reenable it, but there's actually more to it, with ASAN in the context. Agree to remove for now.